easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb

```diff
@@ -0,0 +1,36 @@
+name = 'binutils'
+version = '2.32'
```

Comment:
Any reason for not going with latest binutils  (`2.36`)?

Reply:
Not really, just wanted to stay close to the others in that version range: 8.3, 9.1, 9.2 all use 2.32. I can replace this by 2.36.1 if you think that's ok

Reply:
I see no reason not to go with the latest binutils for a new (minor) GCC release...

Reply:
Done